meta-oe/recipes-support/driverctl/driverctl_git.bb

```diff
@@ -0,0 +1,28 @@
+SUMMARY = "driverctl is a device driver control utility for Linux"
+LICENSE = "LGPLv2.1"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/LGPL-2.1;md5=1a6d268fd218675ffea8be556788b780"
+
+DEPENDS = "systemd"
+RDEPENDS_${PN} += "bash bash-completion"
+
+SRC_URI = " git://gitlab.com/driverctl/driverctl.git"
+SRCREV = "fa9dce43d1a667d6e6e26895fbed01b3b04362c9"
+
+S = "${WORKDIR}/git"
+
+inherit pkgconfig systemd
+
+REQUIRED_DISTRO_FEATURES = "systemd"
+
+FILES_${PN} += "${libdir}"
+FILES_${PN} += "${libdir}/udev"
+FILES_${PN} += "${libdir}/udev/rules.d"
+FILES_${PN} += "${systemd_unitdir}"
+FILES_${PN} += "${systemd_unitdir}/system"
+FILES_${PN} += "${datadir}"
+FILES_${PN} += "${datadir}/bash-completion"
+FILES_${PN} += "${datadir}/bash-completion/completions"
```

Comment:
Please use new style override `FILES:${PN}` here.

Reply:
I did give a try but bitbake failed:

unparsed line: 'FILES:${PN} += "${libdir}"'

Reply:
are you building with master ? if not then please do so

Reply:
Our repo is based on the Duvall branch.

Reply:
update to latest of `dunfell` should fix it too.

Reply:
Tested the new syntax on the latest master branch.